pysmt/solvers/msat.py

```diff
@@ -574,3 +579,81 @@ def declare_variable(self, var):
                                                  tp)
             self.symbol_to_decl[var] = decl
             self.decl_to_symbol[mathsat.msat_decl_id(decl)] = var
+
+
+# Check if we are working on a patched version of the mathsat API
```

Comment:
Since this is part of the official release of mathsat, I think the check is not necessary anymore. We can keep it in order to support older versions of mathsat. In that case, clarify the comment.